docs/contributing/featureflags.md

```diff
@@ -21,15 +21,63 @@ highlights:
 
 ## Using feature flags in code
 
-Default to an "off" state whenever possible – code defensively so that existing functionality is
-maintained should a flag be unavailable altogether. When an interface supports it, also provide
-default values implying "off" to feature flag accessors.
+:::tip
+
+When coding against a feature flag, default to an "off" state whenever possible – code defensively
+so that existing functionality is maintained should a flag be unavailable altogether. When an
+interface supports it, also provide default values implying "off" to feature flag accessors.
 
 Offline mode makes default values even more important, and local development as well as self-hosted
 installations imply being offline. Set a safe default value not just in the flag definition online
 at LaunchDarkly but in code.
 
-### Server
+:::
+
+### Client implementations
+
+#### Web clients
+
+The feature flag values are retrieved through the `fetchServerConfig()` method on the
+`ConfigService`. They are refreshed from the server at the following points in the application
+lifecycle:
+
+- On application startup
+- Every hour after application startup
+- On sync (both automatic and manual)
+
+To use a feature flag, you should first define the new feature flag as an enum value in the
+`FeatureFlags` enum.
+
+Once that is defined, the value can be retrieved by injecting the `ConfigService` and using one of
+the retrieval methods:
+
+- `getFeatureFlagBool()`
+- `getFeatureFlagString()`
+- `getFeatureFlagNumber()`
+
+Each of these methods accept a default value. This value should always default to "off".
```

Comment:
💭 This is repeated a few times and I am wondering what the right level of that is.

Reply:
I agree.  I think the callout above is enough, so I removed this line.